react/SelectBox/SelectBox.jsx

```diff
@@ -1,15 +1,18 @@
-import React from 'react'
+import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import classNames from 'classnames'
 import ReactSelect from 'react-select'
 import styles from './styles.styl'
 import Icon from '../Icon'
 import palette from '../../stylus/settings/palette.json'
+import Overlay from '../Overlay'
+import withBreakpoints from '../helpers/withBreakpoints'
 
 const customStyles = {
   container: base => ({
     ...base,
-    maxWidth: '30rem'
+    maxWidth: '30rem',
+    zIndex: 51
```

Comment:
mmh 51, must be coming from `$overlay-index + 1`, could this be explicited above ?

I think you can even get the value of overlay-index from a CSS variable that could be instantiated from the stylus variable.

https://github.com/css-modules/css-modules/issues/106